types/eyevinn-iaf/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for non-npm package eyevinn-iaf 1.0
+// Project: https://github.com/Eyevinn/ingest-application-framework
+// Definitions by: Eyevinn Technology <https://github.com/Eyevinn>
```

Comment:
this needs to be personal handle, not an  organisation, sorrry!

Reply:
Ok, I've updated it to use personal handles.